governance/proposals/012-set-protocol-fee-cross-bridge.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * This proposal send calls accross the bridge to enable protocol fees
+ * on all supported networks.
+ */
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { Unlock } = require('@unlock-protocol/contracts')
+const { IConnext, targetChains } = require('../helpers/bridge')
+const { ethers } = require('hardhat')
+const PROTOCOL_FEE_IN_BASIS_POINTS = '100' // 1% in basis points
+
+const parseSetProtocolFeeCalls = async (destChainId) => {
+  const { unlockAddress } = await getNetwork(destChainId)
+
+  console.log(`Proposol to set protocolFee to ${ethers.formatEther(protocolFee)}
+    - unlock : ${unlockAddress}`)
+
+  // get Unlock interface
+  const { interface: unlockInterface } = await ethers.getContractAt(
+    Unlock.abi,
+    unlockAddress
+  )
+
+  const calldata = unlockInterface.encodeFunctionData('setProtocolFee', [
+    PROTOCOL_FEE_IN_BASIS_POINTS,
+  ])
+
+  // encode instructions to be executed by the SAFE
+  const moduleData = await ethers.defaultAbiCoder.encode(
```

Comment:
We use ethers 6 here so it will be sth like 

```
const abiCoder = ethers.AbiCoder.defaultAbiCoder()
const moduleData = await abiCoder.encode(

```